sklearn/gaussian_process/kernels.py

```diff
@@ -355,6 +356,10 @@ def diag(self, X):
     def is_stationary(self):
         """Returns whether the kernel is stationary. """
 
+    @abstractmethod
```

Comment:
I'm not entirely comfortable doing this as it will break any custom kernels users have created... they will not be able to construct their kernel.

Could we instead default to true for the sake of backwards compatibility?

Also, should this be a method or a property?

Reply:
I made it defaults to `True`, and removed `VectorKernelMixin` because it is now redundant. I think it is fine either as a method or property, but what confuses me is that `is_stationary` is a method instead of a property. Is there some particular thought behind this?